build.gradle

```diff
@@ -42,6 +54,34 @@ allprojects {
     }
 }
 
+publishing {
+    publications {
+        Main(MavenPublication) {
+            from components.java
+            groupId project.group
+            artifactId 'java-logging'
```

Comment:
does this publish the java-logging-spring etc?

Reply:
added config for subprojects